bom-dependencies/pom.xml

```diff
@@ -279,6 +280,11 @@
                 <artifactId>objenesis</artifactId>
                 <version>${objenesis.version}</version>
             </dependency>
+            <dependency>
+                <groupId>pl.pragmatists</groupId>
+                <artifactId>JUnitParams</artifactId>
+                <version>${junit-params.version}</version>
+            </dependency>
```

Comment:
since no other unit tests use this, i'd prefer to not introduce. it creates a strange style to the codebase, i.e.:

- 1 person uses this
- another uses standard junit
- another wants to introduce TestNG

etc etc etc

It doesnt look like it adds a massive amount of benefit, other than not having to call the method at test start up to get some test data.

Reply:
Style remains the same. It is still JUnit with small extension. This is not a TestNG of even JUnit5.

It simplifies code a lot. Instead of writing tests with names like `testTimestampConversion`, `testSqlDateConversion`, `testUtilDateConversion` and etc. with the same code except one line it can be rewritten in one test with large amount of test cases. All of these 
* `DateConverterTest`
* `IntegerConverterTest`
* `PrimitiveOrWrapperConverterTest`
* `StringConverterTest`

can be replaced with one test and set of different test cases what will reduce amount of boilerplate code in tests and will make addition of new test cases much more easier than now.

Reply:
@vadeg ; sorry, still not convinced. i understand that it reduces code, but it introduces a test dependancy that i've never seen before (i work with a lot of different companies as a job) and only ever used in this 1 class.